end_to_end_tests/golden-record/my_test_api_client/types.py

```diff
@@ -1,8 +1,11 @@
 """ Contains some shared types for properties """
-from typing import BinaryIO, Generic, MutableMapping, Optional, TextIO, Tuple, TypeVar, Union
+from typing import Any, BinaryIO, Generic, MutableMapping, NewType, Optional, TextIO, Tuple, TypeVar, Union
 
 import attr
 
+Unset = NewType("Unset", object)
+UNSET: Any = Unset(object())
```

Comment:
Is the type of this not `Unset`? I haven't actually used `NewType` before.

Reply:
I would say if we can't use `Unset` as an actual type both here and when typing potential `UNSET`s, we should `class Unset:` to make it possible.